### PR TITLE
purge cdn after updateing sotrage account

### PR DESCRIPTION
**Pull request by timothybest-usds**
## Related Issue or Background Info

- Issue https://app.zenhub.com/workspaces/simplereport-5ff8be8854ae8b000e378210/issues/cdcgov/prime-simplereport/990

## Changes Proposed

- purge CDN after uploading new files to the storage account

## Additional Information

- CLI docs https://docs.microsoft.com/en-us/cli/azure/cdn/endpoint?view=azure-cli-latest#az_cdn_endpoint_purge
- Azure CDN info https://docs.microsoft.com/en-us/azure/cdn/cdn-add-to-web-app
- Made the same changes on the public site https://github.com/CDCgov/prime-simplereport/pull/1006

### CODE REVIEW

File: .github/workflows/deployDemo.yml [hunk 1]
```diff
@@ -56,3 +56,5 @@ jobs:
           creds: ${{ secrets.AZURE_CREDENTIALS }}
       - name: Deploy frontend app
         run: yarn run deploy-azure-${{ env.DEPLOY_ENV }}
```
Comment (jeskew-gov) [inline .github/workflows/deployDemo.yml:58]: Just a general question: why are we interpolating the environment name when each environment gets its own workflow definition? I didn't go character by character, but it looks like these workflows vary only by trigger and by the `DEPLOY_ENV` env var.

**Review by jeskew-gov — APPROVED** [review]
Only had one question that is not specific to this PR.

File: .github/workflows/deployDemo.yml (hunk 1 above)
Comment (timothybest-usds) [reply .github/workflows/deployDemo.yml:58]: It was a low effort way for me to copy paste this workflow 7 times

**Comment by benwarfield-usds** [comment]
There's an area that I understand only modestly, would it make sense to only purge the index file, since everything else is a hashed bundle, and then have other things age out?

**Pull request merged**